ModularTegustation/tegu_items/lc13/refinery/crates.dm

```diff
@@ -0,0 +1,118 @@
+		/obj/item/toy/plush/rodion,
+		/obj/item/toy/plush/sinclair,
+		/obj/item/toy/plush/outis,
+		/obj/item/toy/plush/gregor,
+		/obj/item/toy/plush/dante,
+		/obj/item/toy/plush/yuri)
+
+/obj/structure/lootcrate/attackby(obj/item/I, mob/living/user, params)
+	..()
+	if(I.tool_behaviour != TOOL_CROWBAR)
+		return
+	var/loot = pick(lootlist)
+	to_chat(user, "<span class='notice'>You open the crate!</span>")
+	new loot(get_turf(src))
+	qdel(src)
+
+/obj/structure/lootcrate/limbus
+	name = "Limbus Company Crate"
+	desc = "A crate recieved from limbus company. Open with a Crowbar."
+	icon_state = "crate_lcb"
+	lootlist =	list(
+		/obj/item/ego_weapon/mini/hayong,
+		/obj/item/ego_weapon/shield/walpurgisnacht,
+		/obj/item/ego_weapon/lance/suenoimpossible,
+		/obj/item/ego_weapon/shield/sangria,
+		/obj/item/ego_weapon/mini/soleil,
+		/obj/item/ego_weapon/taixuhuanjing,
+		/obj/item/ego_weapon/revenge,
+		/obj/item/ego_weapon/shield/hearse,
+		/obj/item/ego_weapon/mini/hearse,
+		/obj/item/ego_weapon/raskolot,
+		/obj/item/ego_weapon/vogel,
+		/obj/item/ego_weapon/nobody,
+		/obj/item/ego_weapon/ungezifer,
+		/obj/item/clothing/suit/armor/ego_gear/limbus/limbus_coat,
+		/obj/item/clothing/suit/armor/ego_gear/limbus/limbus_coat_short,
+		/obj/item/clothing/under/limbus/shirt,
+		/obj/item/clothing/accessory/limbusvest,
+		/obj/item/clothing/under/limbus/prison,
+		/obj/item/clothing/neck/limbus_tie,)
+
+/obj/structure/lootcrate/k_corp
+	name = "K Corp Crate"
+	desc = "A crate recieved from K-Corp. Open with a Crowbar."
+	icon_state = "crate_kcorp"
+	lootlist =	list(
+		/obj/item/managerbullet,
+		/obj/item/managerbullet/red,
+		/obj/item/managerbullet/white,
+		/obj/item/managerbullet/black,
+		/obj/item/managerbullet/pale,
+		/obj/item/managerbullet/slowdown)
+
+/obj/structure/lootcrate/r_corp
+	name = "R Corp Crate"
+	desc = "A crate recieved from R-Corp. Open with a Crowbar."
+	icon_state = "crate_rcorp"
+	lootlist =	list(
+		/obj/item/ego_weapon/rabbit_blade,
+		/obj/item/gun/energy/e_gun/rabbit,
+		/obj/item/clothing/head/rabbit_helmet,
+		/obj/item/clothing/suit/armor/ego_gear/rabbit/grunts,
+		/obj/item/clothing/under/suit/lobotomy/rabbit,
+		/obj/item/powered_gadget/detector_gadget/ordeal,
+		/obj/item/clothing/suit/space/hardsuit/rabbit,
+		/obj/item/clothing/suit/space/hardsuit/rabbit/leader,
+		/obj/item/toy/plush/myo,
+		/obj/item/toy/plush/rabbit,
+		)
+
+/obj/structure/lootcrate/s_corp
+	name = "S Corp Crate"
+	desc = "A crate recieved from the mysterious S-Corp. Open with a Crowbar."
+	icon_state = "crate_shrimp"
+	lootlist =	list(
+		/obj/item/reagent_containers/food/drinks/soda_cans/wellcheers_red,
+		/obj/item/reagent_containers/food/drinks/soda_cans/wellcheers_white,
+		/obj/item/reagent_containers/food/drinks/soda_cans/wellcheers_purple,
+		/mob/living/simple_animal/hostile/shrimp,
+		/obj/item/gun/ego_gun/sodashotty,
+		/obj/item/gun/ego_gun/sodarifle,
+		/obj/item/gun/ego_gun/sodasmg,
+		/obj/item/grenade/spawnergrenade/shrimp,
```

Comment:
```suggestion
		/obj/item/grenade/spawnergrenade/shrimp
```

Reply:
Makes it harder to add more later, will still fix.